src/helm/benchmark/run.py

```diff
@@ -297,7 +305,8 @@ def main():
         suite=args.suite,
         dry_run=args.dry_run,
         skip_instances=args.skip_instances,
-        save_instances_only=args.save_instances_only,
+        write_instances=args.write_instances,
+        write_instances_only=args.write_instances_only,
         read_instances_from_file=args.read_instances_from_file,
```

Comment:
I'm still not sure what the right flags should be. I wonder if we should just have two:

- `cache_instances`: write scenarios if doesn't exist, read them otherwise.
- `cache_instances_only`: exit after caching instances

Then:

- In the scenario instance caching bundle, we run with both flags
- In the helm-run bundle, we run with only the first flag
- When running locally, neither flag is set (so there is no scenario instance cache)

What do you think?

Reply:
Done. I like this; it feels like the least clunky solution.